server/community.go

```diff
@@ -64,3 +66,62 @@ func (s *Server) postPRWelcomeMessage(ctx context.Context, pr *model.PullRequest
 
 	return nil
 }
+
+func (s *Server) assignGreeter(ctx context.Context, pr *model.PullRequest, repo *Repository) error {
+	// Only assign an Usher for community PRs
+	if s.IsOrgMember(pr.Username) {
+		return nil
+	}
+
+	// Is it configured to have a greeting team to handle the PR?
```

Comment:
Should we test this first to avoid unnecessary GitHub client API calls for `IsOrgMember` when a repo has no such configuration?

Reply:
IsOrgMember fetches list of org members in a CRON once per day, so this shouldn't be an issue.